ansible/roles/bastion/tasks/main.yml

```diff
@@ -34,8 +34,11 @@
     - copy_env_private_key
 
 - name: Generate host .ssh/config Template
-  become: no
-  local_action: template src={{ role_path }}/templates/bastion_ssh_config.j2 dest={{output_dir}}/ssh-config-{{ env_type }}-{{ guid }}
+  become: false
+  template:
+    src: "{{ role_path }}/templates/bastion_ssh_config.j2"
```

Comment:
Let's fix the source path for the template. `{{ role_path }}/templates/` is the standard lookup path for template files. So this can be:

```
  template:
￼    src: bastion_ssh_config.j2
```